django_nextjs/render.py

```diff
@@ -129,6 +152,7 @@ async def render_nextjs_page(
         using=using,
         allow_redirects=allow_redirects,
         headers=headers,
+        include_content_type=content_type is None,
```

Comment:
I suggest we delete `include_content_type` and pass the "Content-Type" header of incoming request without any condition to the Next.js server.

Because `content_type` in here exists only for `HttpResponse`, same as `django.shortcuts.render` function.

What do you think?

Reply:
Hi, thanks for the excellent library! 

I just want to double check that I'm understanding you correctly. 

My reasoning for adding the `include_content_type` was so that `render_nextjs_page`'s `content_type` kwarg acts like an override for whatever `Content-Type` header NextJS returns. I'm not sure if this scenario really makes sense, but I thought for the sake of backwards compatibility it should still be respected.

Are you suggesting that the `Content-Type` returned by NextJS should be unconditionally used? In that case should the `content_type` kwarg be removed from the `render_nextjs_page` function? 

I'm not entirely sure in what scenario the `content_type` kwarg would be used (I don't use it at all in my application) so I figured this way would cause the least friction for anyone who does use it.

Reply:
Perhaps a cleaner way for me to have written it would be for `_render_nextjs_page_to_string` to also return the response's content-type and then apply the  `content_type` kwarg as an override similar to how `override_status` works. Something like this:
```python
    content, status, response_headers, response_content_type = await _render_nextjs_page_to_string(
        request,
        template_name,
        context,
        using=using,
        allow_redirects=allow_redirects,
        headers=headers,
        include_content_type=content_type is None,
    )
    final_status = status if override_status is None else override_status
    final_content_type = response_content_type if content_type is None else content_type
    return HttpResponse(content, final_content_type, final_status, headers=response_headers)
```
Let me know if you want me to make that change :smile:

Reply:
Makes sense. I've mismatched the Nextjs response headers and request headers.